contrib/win32/install/client.wxs

```diff
@@ -0,0 +1,29 @@
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Fragment>
+        <!-- KeyPath is necessary for multi-file components to identify the key file - preferrably versioned. -->
+        <ComponentGroup Id="Client" Directory="INSTALLFOLDER">
+            <ComponentGroupRef Id="Shared" />
+            <Component>
+                <File Name="ssh.exe" KeyPath="yes" />
+                <File Name="ssh.pdb" />
+            </Component>
+            <Component>
+                <File Name="sftp.exe" KeyPath="yes" />
+                <File Name="sftp.pdb" />
+            </Component>
+            <Component>
+                <File Name="ssh-add.exe" KeyPath="yes" />
+                <File Name="ssh-add.pdb" />
+            </Component>
+            <Component>
+                <File Name="ssh-keyscan.exe" KeyPath="yes" />
```

Comment:
ssh-agent.exe is also part of client

Reply:
It's already pulled in via the shared fragment and will be in both: https://github.com/PowerShell/openssh-portable/pull/521/files/b4d18222ba33955720eca7e7112c4796d392cb49#diff-a5dc61629f16fdc0a8c04f725ac9657edc9be2efb588fa7b700fd1049e2c3858R29

MSI best practices are general one (preferably versioned) file per component.